packages/ember-glimmer/lib/component.js

```diff
@@ -85,7 +85,7 @@ export const BOUNDS = symbol('BOUNDS');
   `hello` for the `app-profile` component:
 
   ```javascript
-  App.AppProfileComponent = Ember.Component.extend({
```

Comment:
API docs support paths, so you can do something like the following:

    ```app/components/profile.hbs
    import Ember from 'ember';
  
    export default Ember.Component.extend({
    …
    });
    ```

The idea is to provide as full a file as possible to try to make it less confusing. The paths should use the classic structure, not pods, and imports should be added. This last point will hopefully help users if the JavaScript module API RFC gets greenlit.